README.md

```diff
@@ -0,0 +1,18 @@
+# DANDI Handbook
+Handbook for interacting with the DANDI archive.
+
+## DANDI Style Guidelines
+
+Follow the guidelines below when creating and revising text in the DANDI Handbook:
+
+*	**dandi-** repositories - hyphenate the names of DANDI GitHub repositories (e.g. **dandi-archive**); Dandisets is an exception because it is a
+complete word
+*	**Dandiset** (**not** dandiset)
+*	file names - use lower case (e.g. **development.md**)
+*	headings - use Title Capitalization (for 1st and 2nd levels) and follow with an intro sentence
+*	**GitHub** camel case (**not** github or Github)
+*	instructional language should be direct, imperative, active, straightforward (e.g. "Install the files in your Python environment" *not* "Files could be installed in your Python environment")
```

Comment:
```suggestion
*	Instructional language should be direct, imperative, active, straightforward (e.g. "Install the files in your Python environment", *not* "Files could be installed in your Python environment")
```

Reply:
not capitalizing the first word in these bulleted phrases

Reply:
```suggestion
*	instructional language should be direct, imperative, active, straightforward (e.g. "Install the files in your Python environment", *not* "Files could be installed in your Python environment")
```